src/Serial.c

```diff
@@ -0,0 +1,64 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "Serial.h"
+
+#include <uart_if.h>
+
+#include "Task.h"
+
+static void
+Run(struct SerialOutTask* aSerialOut)
+{
+  InitTerm();
+  ClearTerm();
+
+  for (;;) {
+    struct IPCMessage msg;
```

Comment:
for example (about the struct thing), I'd used to seeing ```IPCMessage msg;``` without the struct here.